app/main.py

```diff
@@ -1,40 +1,40 @@
 class Car:
-    def __init__(self, comfort_class, clean_mark, brand):
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
         self.comfort_class = comfort_class
         self.clean_mark = clean_mark
         self.brand = brand
 
 
 class CarWashStation:
     def __init__(self,
-                 distance_from_city_center,
-                 clean_power,
-                 average_rating,
-                 count_of_ratings):
+                 distance_from_city_center: int,
```

Comment:
Check is it int or float
And also add type annotation for constructor and all methods. What they return. Like so __init(.....) -> None: